generation/gen-big5-test.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
```

Comment:
nit: maybe merge this file with gen-gbk-test? They look pretty similar. We could name the merged file smth. like `gen-gbk-big5-fixtures.js`.